mlir/include/mlir/Dialect/Bufferization/IR/BufferizableOpInterface.h

```diff
@@ -261,9 +260,9 @@ struct BufferizationOptions {
   using AnalysisStateInitFn = std::function<void(AnalysisState &)>;
   /// Tensor -> MemRef type converter.
   /// Parameters: Value, memory space, func op, bufferization options
-  using FunctionArgTypeConverterFn = std::function<BaseMemRefType(
-      TensorType, Attribute memorySpace, FunctionOpInterface,
-      const BufferizationOptions &)>;
+  using FunctionArgTypeConverterFn =
```

Comment:
It looks like there are some unrelated bufferization changes in this PR.

Reply:
Sorry about that, wrong base branch 🤦🏻 This is now fixed.